ride_share.rb

```diff
@@ -0,0 +1,191 @@
+#Developer's name = Tithvorlak Mok (Lak)
+#Project: Ride Share
+#Due Date: 02/10/2020
+
+#Data of Rides
+total_rides = {
+    DR0001: [
+        {
+            date: '3rd Feb 2016',
+            cost: 10,
+            rider_id: 'RD0003',
+            rating: 3
+        },
+        {
```

Comment:
Looks like you might be using 4 spaces per tab in your text editor rather than 2? You can adjust this in the settings, and I really recommend it because it makes the code look a lot more connected.